adding note about persistent connections for L7 log generation (#510)
tigera/docs: calico-enterprise/visibility/elastic/l7/configure.mdx

   kubectl annotate svc <service-name> -n <service-namespace> projectcalico.org/l7-logging-
   ```

> After annotating a service for L7 log collection, only newly established connections through that service will be proxied by Envoy, connections established before the service was annotated will not be proxied or interrupted, resulting in no log generation.

Reviewer: ```suggestion
After annotating a service for L7 log collection, only newly-established connections through that service are proxied by Envoy. Connections established before the service is annotated are not proxied or interrupted, and no logs are generated.
```

> Conversely, when a service is deselected, any previous connections established through the annotated service will continue to be proxied by Envoy until they are terminated and will generate logs.

Reviewer: ```suggestion
Conversely, when a service is deselected, any previous connections established through the annotated service continue to be proxied by Envoy until they are terminated, and logs are generated.
```
